catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/model/FilterNodeMapImpl.java

```diff
@@ -100,6 +102,24 @@ public String getValue() {
     return Objects.toString(json.get(VALUE));
   }
 
+  @Override
+  public Double getDistance() {
+    if (!isLeaf()) {
+      throw new IllegalStateException("No distance value exists for a logical operator");
+    }
+
+    Object distance = json.get(DISTANCE);
+    if (distance == null) {
+      return null;
+    }
+
+    if (distance instanceof Number) {
```

Comment:
✏️ This can be part of the conditional above as an `else if`

Reply:
I prefer to keep it as its own `if`